package.json

```diff
@@ -87,9 +87,6 @@
     "webpack-cli": "^4.3.1",
     "whatwg-fetch": "^3.5.0"
   },
-  "engines": {
-    "node": "14.4.*"
```

Comment:
what a jerk, good riddance.

Reply:
What version of Node will it uses after removing this part from the package.json? Will it always be the latest?

Reply:
hm, good point. Whatever it's set to actually install, so for docker that's currently 14.13.1, for github actions it's "latest version of 14", but heroku prefers it's prespecified so I guess I'll set this to `>=14.4` so that v15 doesn't break things.